<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Source_window</class>
 <widget class="QMainWindow" name="Source_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="head_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Unit Conventer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="len_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>len.jpg</normaloff>len.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>135</width>
      <height>135</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="temp_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>temp.jpg</normaloff>temp.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>105</width>
      <height>105</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="area_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>area.jpg</normaloff>area.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="vol_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>volume.jpg</normaloff>volume.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>115</width>
      <height>115</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="wei_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>weight.jpg</normaloff>weight.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>115</width>
      <height>115</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="time_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>time.jpg</normaloff>time.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>130</width>
      <height>130</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 208, 255);</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
